draft-ietf-quic-transport.md

```diff
@@ -2375,8 +2376,11 @@ Unlike TCP SACKs, QUIC acknowledgements are irrevocable.  Once a packet has
 been acknowledged, even if it does not appear in a future ACK frame,
 it remains acknowledged.
 
-A client MUST NOT acknowledge Version Negotiation or Retry packets.  These
-packet types contain packet numbers selected by the client, not the server.
+A client MUST NOT acknowledge Version Negotiation or Retry packets.  Version
```

Comment:
nit: technically they CAN NOT acknowledge VN packets, since there's no packet number.

How about "A client MUST NOT acknowledge Retry packets.  Version Negotiation packets cannot be acknowledged because they do not contain a packet number."